search/search_repository.go

```diff
@@ -568,6 +568,17 @@ func (r *GormSearchRepository) listItemsFromDB(ctx context.Context, criteria cri
 		return nil, 0, errors.NewBadParameterError("expression", criteria)
 	}
 
+	if parentExists != nil && !*parentExists {
+		where += ` AND
+			id not in (
+				SELECT target_id FROM work_item_links
+				WHERE link_type_id IN (
+					SELECT id FROM work_item_link_types WHERE forward_name = 'parent of'
```

Comment:
@baijum Is there a need to add `forward_name = 'parent of'`?
This would work even if you replace line number 576 with `SELECT id FROM work_item_link_types`.